stellar-common/src/payments.rs

```diff
@@ -50,6 +70,7 @@ fn map_payments(block: Block) -> Result<Payments, substreams::errors::Error> {
                         Some(account) => account.to_string(),
 
                         None => {
+                            // Similarly here: the sender will always be present.
```

Comment:
Let me check, but I am sure to have found some strange operations where there was no sender or something like that

Reply:
Ok here is an example of what I mean: https://stellar.expert/explorer/testnet/tx/f3d3df2afb85f5a93b0485ebe5c243033e066e8b654de79fd7c9fd88a8c726b4

Here if I take the source account from the transaction it is not the same as the source account shown in the above link.

Reply:
Ah I see what you mean. There is a hierarchy to sources: an operation can have a source account which is different than the transaction in which it's bundled's source account. If it's omitted the tx source is implied, so you need to check both.